src/Exception/TemplateRenderException.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Dhii\Output\Exception;
+
+use Exception as RootException;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Psr\Container\ContainerInterface;
+use Dhii\Output\TemplateInterface;
+use Dhii\Output\RendererAwareTrait;
+use Dhii\Output\ContextAwareTrait;
+use Dhii\I18n\StringTranslatingTrait;
+use Dhii\Exception\CreateInvalidArgumentExceptionCapableTrait;
+
+/**
+ * An exception that occurs when a renderer cannot produce output.
```

Comment:
Description looks like an elusive copy-pasta rogue.

> An exception that occurs in relation to a template.

Reply:
Not quite the right alternative, but point taken.